plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -152,3 +173,29 @@ func rename(attrs pcommon.Map, renameMap map[string]string) {
 		}
 	}
 }
+
+func truncateAttributesByLength(attributes pcommon.Map) {
+	for attrKey, _ := range attributesRenamingForMetric {
```

Comment:
nit: Don't need to include the value.
```suggestion
	for attrKey := range attributesRenamingForMetric {
```

Reply:
Removed.